.github/workflows/python-package.yml

```diff
@@ -131,6 +131,36 @@ jobs:
         name: py-opentimelineio-codecov
         fail_ci_if_error: true
 
+  py_build_test_mingw64:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: 'recursive'
+    - name: Setup MSYS2
+      uses: msys2/setup-msys2@v2
+      with:
+        msystem: mingw64
+        install: >-
+          mingw-w64-x86_64-python
+          mingw-w64-x86_64-python-setuptools
```

Comment:
Only python, pip, make and GCC should be required. Like that. `build` can be installed with `python -m pip install build` (which I trust more i think than the one from mingw) and the rest is defined in the `pyproject.toml` and will be installed in a virtualenv by `build`.

Reply:
Thanks for taking a look at this! I'll take a look at reducing the these dependencies.